src/api.rs

```diff
@@ -1,2 +1,8 @@
+//! This module contains all the APIs that are exposed through javascript
```

Comment:
```suggestion
//! All the APIs that are exposed through JavaScript
```

Just to run with @blaumeise20's thought.  He might have additional suggestions.